android/app/build.gradle

```diff
@@ -64,12 +64,11 @@ android {
     buildTypes {
         debug {
             // app id will be "de.janoodle.circlesApp.debug"
-            applicationIdSuffix ".debug"
         }
         release {
             // NOTE: ProGuard is not enabled at the moment
             // See: https://flutter.dev/docs/deployment/android to enable it
-            signingConfig signingConfigs.debug
+            // signingConfig signingConfigs.release
```

Comment:
❓ why was this necessary?

Reply:
No signing configurations for a debug build but there is one for a release build. From line 55
```
//     signingConfigs {
//         release {
//             keyAlias keystoreProperties['keyAlias']
//             keyPassword keystoreProperties['keyPassword']
//             storeFile file(keystoreProperties['storeFile'])
//             storePassword keystoreProperties['storePassword']
//         }
//     }
```
Should this be ever required, one would simply uncomment and carry on

Reply:
I think that `signingConfigs.debug` is the default one, I would prefer to keep that one.

That's what the new Flutter project template uses.

I'd prefer to restore back the `signingConfig signingConfigs.debug` then.